fred2/missionsave.cpp

```diff
@@ -2672,6 +2672,14 @@ int CFred_mission_save::save_objects() {
 		parse_comments(z ? 2 : 1);
 		fout(" %s\t\t;! Object #%d\n", shipp->ship_name, i);
 
+		// Display name
+		if (save_format != MissionFormat::RETAIL && shipp->has_display_name()) {
```

Comment:
`save_format` and `MissionFormat` aren't declared here. They only exist in qtFRED.